packages/riipen-ui/src/utils/withClasses.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import clsx from "clsx";
+import PropTypes from "prop-types";
+
+const withClasses = () => Component => {
+  if (process.env.NODE_ENV !== "production") {
+    if (Component === undefined) {
+      throw new Error(
+        [
+          "You are calling withStyles(styles)(Component) with an undefined component.",
+          "You may have forgotten to import it."
+        ].join("\n")
+      );
+    }
+  }
+
+  const WithClasses = React.forwardRef(function WithClasses(props, ref) {
+    const { classes, ...other } = props;
+    return (
+      <Component ref={ref} classes={[clsx("riipen", classes)]} {...other} />
```

Comment:
can we do `classes={["riipen", ...classes"]}` instead? I think keeping the passed in classes array mostly intact would be slightly preferable. Just in case somebody wants to do some crazy classes array shenanigans in the future.. It might be unexpected to see a single concatenated classname in that prop 🤷‍♂.

If you decided to make that change also add a defaultProp with `classes: []`!